eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -9,8 +9,7 @@ Param (
 . (Join-Path $PSScriptRoot common.ps1)
 $allPackageProperties = Get-AllPkgProperties $serviceDirectory
 if ($allPackageProperties)
-{
-    New-Item -ItemType Directory -Force -Path $outDirectory
+{ 
```

Comment:
Lets be sure to make these changes in the tools repo.

Reply:
Making common changes in this PR only to test it. I will submit PR to tools repo once tested.